react-frontend/src/components/Nav/footer.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import '../../css/footer.css';
+import { Link } from 'react-router-dom';
 function Footer() {
   return (
-    <footer class="footer">
-      <div class="container">
+    <footer className="footer">
+      <div className="container">
```

Comment:
is this container class used for anything?  wondering if we need it at all or if the footer does the job?